breakpad-symbols/src/sym_file/mod.rs

```diff
@@ -374,24 +418,21 @@ mod test {
     #[test]
     fn test_symbolfile_from_bytes_with_lf() {
         test_symbolfile_from_bytes(
-            b"MODULE Linux x86 ffff0000 bar
-FILE 53 bar.c
-PUBLIC 1234 10 some public
-FUNC 1000 30 10 another func
-1000 30 7 53
-",
+            b"MODULE Linux x86 ffff0000 bar\nFILE 53 bar.c\nPUBLIC 1234 10 some public\nFUNC 1000 30 10 another func\n1000 30 7 53\n",
         );
     }
 
     #[test]
     fn test_symbolfile_from_bytes_with_crlf() {
         test_symbolfile_from_bytes(
-            b"MODULE Linux x86 ffff0000 bar
-FILE 53 bar.c
-PUBLIC 1234 10 some public
-FUNC 1000 30 10 another func
-1000 30 7 53
-",
+            b"MODULE Linux x86 ffff0000 bar\r\nFILE 53 bar.c\r\nPUBLIC 1234 10 some public\r\nFUNC 1000 30 10 another func\r\n1000 30 7 53\r\n",
+        );
+    }
+
+    #[test]
+    fn test_symbolfile_no_trailing_newline() {
+        test_symbolfile_from_bytes(
+            b"MODULE Linux x86 ffff0000 bar\nFILE 53 bar.c\nPUBLIC 1234 10 some public\nFUNC 1000 30 10 another func\n1000 30 7 53",
```

Comment:
the final line here is malformed, right? Can we have a test where the final line is actually good and make sure the parser properly read it?

Reply:
I think the last line is fine, its the same as in the tests above.